### PR TITLE
fix tianocore and apu1 build

### PR DESCRIPTION
**Pull request by piotr-kleins**
Signed-off-by: Piotr Kleinschmidt <piotr.kleins@gmail.com>

### CODE REVIEW

File: src/arch/x86/Makefile.inc [hunk 1]
```diff
@@ -275,6 +275,7 @@ postcar-$(CONFIG_EARLY_EBDA_INIT) += ebda.c
 postcar-$(CONFIG_IDT_IN_EVERY_STAGE) += exception.c
 postcar-$(CONFIG_IDT_IN_EVERY_STAGE) += idt.S
 postcar-y += exit_car.S
+postcar-y += ../../mainboard/$(MAINBOARDDIR)/bios_knobs.c
```
Comment (miczyg1) [inline src/arch/x86/Makefile.inc]: @piotr-kleins why here? add this line in mainboard/pcengines/apu1/Makefile.inc

File: src/drivers/uart/uart8250io.c [hunk 2]
```diff
@@ -21,7 +21,12 @@
 #include <console/uart.h>
 #include <trace.h>
 #include "uart8250reg.h"
-#include "mainboard/pcengines/apu2/bios_knobs.h"
+
+#if CONFIG(BOARD_PCENGINES_APU1)
+	#include "mainboard/pcengines/apu1/bios_knobs.h"
```
Comment (miczyg1) [inline src/drivers/uart/uart8250io.c]: @piotr-kleins no indents for macros

File: src/drivers/uart/uart8250io.c [hunk 3]
```diff
@@ -21,7 +21,12 @@
...
+#else
+	#include "mainboard/pcengines/apu2/bios_knobs.h"
```
Comment (miczyg1) [inline src/drivers/uart/uart8250io.c]: a/a

File: src/arch/x86/Makefile.inc (hunk 1 above)
Comment (piotr-kleins) [reply src/arch/x86/Makefile.inc]: Fixed

File: src/drivers/uart/uart8250io.c (hunk 2 above)
Comment (piotr-kleins) [reply src/drivers/uart/uart8250io.c]: Fixed

File: src/drivers/uart/uart8250io.c (hunk 3 above)
Comment (piotr-kleins) [reply src/drivers/uart/uart8250io.c]: Fixed

**Pull request merged**